qemu/tests/qemu_guest_agent.py

```diff
@@ -820,6 +851,24 @@ class QemuGuestAgentBasicCheckWin(QemuGuestAgentBasicCheck):
     """
     Qemu guest agent test class for windows guest.
     """
+
+    @error_context.context_aware
+    def gagent_uninstall(self, session, vm, *args):
+        if args and isinstance(args, tuple):
+            gagent_uninstall_cmd = args[0]
+        else:
+            raise error.TestError("Missing config 'gagent_uninstall_cmd'")
+
+        if not gagent_uninstall_cmd:
+            return
+
+        error_context.context("Try to uninstall 'qemu-guest-agent' package.",
+                              logging.info)
+        s, o = session.cmd_status_output(gagent_uninstall_cmd)
```

Comment:
Better to use timeout in the session.cmd for windows guest test, i.e.:
s, o = session.cmd_status_output(gagent_uninstall_cmd, timeout=360)

Reply:
Timeout is the default parameter in cmd_status_output function,and the default value is 60 which is enough.